### PR TITLE
Bump velero version v1.3.0 and plugin version v1.0.1

### PR DESCRIPTION
**Pull request by jenting**
Signed-off-by: JenTing Hsiao <jenting.hsiao@suse.com>

### CODE REVIEW

File: charts/velero/README.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ helm upgrade <RELEASE NAME> --set initContainers.backupStorageLocation.name=aws,
 
 ## Upgrading
 
+### Upgrading to v1.3.0
+
+The [instructions found here](https://velero.io/docs/v1.3.0/upgrade-to-1.3/) will assist you in upgrading from version v1.0.0 or v1.1.0 or v1.2.0 to v1.3.0.
```
Comment (nrb) [inline charts/velero/README.md]: I think the upgrade documentation for v1.3.0 only goes from v1.2.0 to v1.3.0. We haven't tested v1.0.0 -> v1.3.0 or v1.1.0 -> v1.3.0

**Comment by nrb** [comment]
Hmm, for some reason the lint failure didn't appear to pull down the AWS plugin image.

File: charts/velero/README.md (hunk 1 above)
Comment (jenting) [reply charts/velero/README.md]: Updated!

**Comment by jenting** [comment]
@nrb CI pass, PTAL if you have chance.

File: charts/velero/README.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ Velero has two main components: a CLI, and a server-side Kubernetes deployment.
 
 ## Installing the Velero CLI
 
-See the different options for installing the [Velero CLI](https://velero.io/docs/v1.2.0/install-overview/#install-the-cli).
+See the different options for installing the [Velero CLI](https://velero.io/docs/v1.3.0/install-overview/#install-the-cli).
```
Comment (skriss) [inline charts/velero/README.md]: This link is no longer valid due to some docs reorg, needs to be `https://velero.io/docs/v1.3.0/basic-install/#install-the-cli`

File: charts/velero/README.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ When installing using the Helm chart, the provider's credential information will
 
 The default configuration values for this chart are listed in values.yaml.
 
-See Velero's full [official documentation](https://velero.io/docs/v1.2.0/install-overview/). More specifically, find your provider in the Velero list of [supported providers](https://velero.io/docs/v1.2.0/supported-providers/) for specific configuration information and examples.
+See Velero's full [official documentation](https://velero.io/docs/v1.3.0/install-overview/). More specifically, find your provider in the Velero list of [supported providers](https://velero.io/docs/v1.3.0/supported-providers/) for specific configuration information and examples.
```
Comment (skriss) [inline charts/velero/README.md]: Update link to `https://velero.io/docs/v1.3.0/basic-install/`

**Pull request merged**